pages/api/roles/world/index.ts

```diff
@@ -1,5 +1,105 @@
+import { PrismaClient, Prisma, RoleType } from '@prisma/client';
 import type { NextApiRequest, NextApiResponse } from 'next';
+import { ApiResponse } from '../../../../types/apiResponse';
+import { WorldRoleListData, WorldRoleQueryParams } from '../../../../types/role';
+import { getNonEmptyArrayOrUndefined, combineDefinedArrays, buildFrequencyMap } from '../../../../utils/arrays';
+import { withVerifiedUser } from '../../../../utils/auth/jwtHelpers';
+import {
+  makeCompanyNameFilters,
+  makeRoleTitleFilters,
+  makeRoleYearFilters,
+  makeRoleTypeFilters,
+} from '../../../../utils/filters/filterHelpers';
+import {
+  convertQueryParamToStringArray,
+  createJsonResponse,
+  HttpMethod,
+  HttpStatus,
+  rejectHttpMethod,
+} from '../../../../utils/http/httpHelpers';
+import { withPrismaErrorHandling } from '../../../../utils/prisma/prismaHelpers';
+import { splitByWhitespaces, splitByCommaRemovingWhitespacesAround } from '../../../../utils/strings/formatters';
 
-export default function handler(req: NextApiRequest, res: NextApiResponse) {
-  res.status(200).end(`Request sent to ${req.url}\n`);
+const prisma = new PrismaClient();
+
+async function handler(userId: string, req: NextApiRequest, res: NextApiResponse) {
+  const method = req.method;
+  switch (method) {
```

Comment:
Super nit (sorry)
```suggestion
  switch (req.method) {
```

Reply:
I think this is fine, since we are also using `method` to pass to `rejectHttpMethod`, rather than doing `req.method` at both places.